perf/perf_hv_gpci.py

```diff
@@ -65,14 +65,17 @@ def setUp(self):
             line = line.split(',')[0].split('/')[1]
             self.list_of_hv_gpci_events.append(line)
 
-        # Clear the dmesg, by that we can capture the delta at the end of the test.
+        # Clear the dmesg, by that we can capture the delta at the end of
+        # the test.
         output = process.run("dmesg -c")
 
     def error_check(self):
         if len(self.fail_cmd) > 0:
             for cmd in range(len(self.fail_cmd)):
-                self.log.info("Failed command: %s" % self.fail_cmd[cmd])
-            self.fail("perf_raw_events: some of the events failed, refer to log")
+                self.log.info("Failed command: %s"
+                              % self.fail_cmd[cmd])
```

Comment:
Any reason for this line change? I see it is already within the 80 character limit.

Reply:
ah autopep8 does that I think no harm if have this way

Reply:
Do you mean change is okay with no specific reason? I feel it is unnecessary as it does not add value.

Reply:
No i am saying pep8 commit just included this cahnge  but it has other changes as well if you say i can change this will do so

Reply:
other changes are fine, just wanted this line change to be removed as it there is no new value to it.